.rubocop_todo.yml

```diff
@@ -267,12 +269,16 @@ Naming/FileName:
 Naming/HeredocDelimiterCase:
   Enabled: false
 
-# Offense count: 20
+# Offense count: 23
 # Configuration parameters: Blacklist.
 # Blacklist: END, (?-mix:EO[A-Z]{1})
 Naming/HeredocDelimiterNaming:
   Enabled: false
 
+# Offense count: 19
+Naming/MemoizedInstanceVariableName:
```

Comment:
should we add new issues in redmine for new cops?

Reply:
I would say only if one does open a PR for fixing.